app/assets/javascripts/discourse/app/services/user-tips.js

```diff
@@ -95,7 +75,7 @@ export default class UserTips extends Service {
   }
 
   hideTip(userTipId, force = false) {
-    // Tippy instances are not destroyed immediately because sometimes there
+    // Instances are not destroyed immediately because sometimes there
```

Comment:
a (preexisting) typo, I think?

```suggestion
    // Instances are not destroyed immediately because sometimes their
```